src/libs/antares/study/parts/common/cluster.cpp

```diff
@@ -30,7 +30,7 @@ namespace Antares::Data
 {
 Cluster::Cluster(Area* parent) :
     parentArea(parent),
-    series(tsNumbers)
+    series(tsNumbers, "thermal-availability")
```

Comment:
As recommended in https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2073 (please see https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2073#discussion_r1591156960), we should not have to make this change